src/@next/Radio/index.tsx

```diff
@@ -0,0 +1 @@
+export { Radio } from './Radio';
```

Comment:
can you follow the pattern on how we export in the index?

Reply:
Fixed